mllib/src/test/scala/org/apache/spark/ml/classification/GBTClassifierSuite.scala

```diff
@@ -119,6 +119,31 @@ class GBTClassifierSuite extends SparkFunSuite with MLlibTestSparkContext {
   */
 
   /////////////////////////////////////////////////////////////////////////////
+  // Tests of feature importance
+  /////////////////////////////////////////////////////////////////////////////
+  test("Feature importance with toy data") {
```

Comment:
I have some concerns about the number of times this test is copy/pasted in the testing suites currently. I don't have a good solution for it at the moment, however. I think this will need to be addressed in the future since more and more models may have feature importances and will need to do essentially the same test. I think we could open another JIRA to improve feature importance tests in ML. Thoughts?